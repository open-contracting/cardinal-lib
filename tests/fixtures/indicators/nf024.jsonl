{"ocid":"0","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":1,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}]}
{"ocid":"1","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":101,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}]}
{"ocid":"2","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":101,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}]}
{"ocid":"3","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":101,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}]}
{"ocid":"accept","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":101,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]},{"status":"cancelled","suppliers":[{"id":"C"}]},{"status":"unsuccessful","suppliers":[{"id":"U"}]}]}
{"ocid":"pending","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":1,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]},{"status":"pending"}]}
{"ocid":"invalid","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":1,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]},{"status":"invalid"}]}
{"ocid":"nonprice","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":2,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":1,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}]}
{"ocid":"multiple awards","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":1,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]},{"status":"active","suppliers":[{"id":"W"}]}]}
{"ocid":"multiple tenderers","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"},{"id":"M"}],"value":{"amount":1,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}]}
{"ocid":"multiple suppliers","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":1,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"},{"id":"X"}]}]}
