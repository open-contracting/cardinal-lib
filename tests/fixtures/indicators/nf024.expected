{"0":{"NF024":0.0}}
